mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include "mom_stickybomb.h"
```

Comment:
Use forward declaration instead of including this here if possible. May require changing the utlvector to be of sticky bomb pointers